examples/receive-audio-through-aic-filter/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "example-receive-audio-through-aic-filter",
+  "author": "LiveKit",
+  "private": true,
+  "description": "",
+  "type": "module",
+  "main": "index.ts",
+  "scripts": {
+    "build": "tsc --incremental",
+    "lint": "eslint -f unix \"**/*.ts\""
+  },
+  "keywords": [],
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@livekit/audio-filter-uniffi": "link:/Users/ryan/w/livekit/aic-noise-filter/target/packages/node",
```

Comment:
todo: This needs to be updated once the aic audio filter package gets published, and is what seems to be keeping teh ci build from passing at the moment.